PostHog/posthog
# fix: basic insights including derived names

## Problem

Some uses of "basic serialized" insights assume they have the derived name available.
 But they don't...

## Changes

Now they do!

## How did you test this code?

* Making an insight with a derived name
* look at the home page and see it is shown as "Insight"
* change the API response
* see that it has its derived name
